lib/iceberg/events/life_stop.js

```diff
@@ -9,10 +9,16 @@
  * @param {AOInstance} instance - AO instance
  */
 const onLifeStop = async (instance = {}) => {
-  const { h = {} } = instance
-  const { debouncedSubmitOrders } = h
+  const { state = {}, h = {} } = instance
+  const { args = {}, orders = {}, gid } = state
+  const { debouncedSubmitOrders, emit, debug } = h
+  const { cancelDelay } = args
 
   debouncedSubmitOrders.cancel()
+
+  debug('detected iceberg algo cancelation, stopping...')
+
+  await emit('exec:order:cancel:all', gid, orders, cancelDelay)
```

Comment:
should happen immediately, not with delay?

Reply:
set to `0`?

Reply:
hmmm, its a "feature" right now... no idea what the sense of it is

Reply:
`onLifeStop ` should have a cancel delay of 0, when we want to keep the feature, we need the separate cancel in `orders_order_cancel.js`